### DISCUSSION TITLE
Multiple "company" names

### DISCUSSION THREAD

**Discussion by jeffreytull** [snipe/snipe-it#11494] [Q&A]
I'm investigating using Snipe-IT for device management and tracking for my school district.  We are spread out over 5 buildings plus central office.  Has anyone been able to add Location to asset labels?  Company name is not enough; we want the label to show where the device should be going.  I read another post that custom labels is not yet an option, and I'm okay with not being able to customize.  I just want to add School Name as well as district name (i.e. company name).

Thanks in advance. =)

**Comment by jeffreytull** [comment]
Actually found the workaround myself.  Additional Companies can be added.... so putting all the schools as "companies" will allow us to identify where devices go on the label.  Yay!!  Hope this helps somebody someday.

**Comment by mikeroq** [comment]
My work around on that is I print my labels outside snipeit. For instance I have a Brother QL-500 label printer. I made a design in the P-Touch software and used a CSV export of the data I needed to print the labels.
